vtex/loaders/legacy/productList.ts

```diff
@@ -88,10 +102,24 @@ const fromProps = (
     O?: LegacySort;
   };
 
+  if (isSKUIDProps(props)) {
+    const ids = props.ids || [];
+    const skus = props.skus || [];
+
+    const skuIds = new Set([...ids, ...skus]);
+    Array.from(skuIds).forEach((skuId) => params.fq.push(`skuId:${skuId}`));
```

Comment:
Why are you passing the `ids` and `skus`? Don't they represent different things?

Reply:
I added the additional `skus` prop due to backward compatibility concerns but removed it on [250b1c4](https://github.com/deco-cx/apps/pull/46/commits/250b1c4089b597066530e02f7302210a5eef9631). I kept the original `ids` prop and improved the comments and typing.